nasademo/src/main/resources/db/migration/V1_addCameraTable.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS cameras(
+    id BIGINT not null,
```

Comment:
Hi! Primary key by default has not null and unique constraint, so there is no need to add this here